benchmarks/docker/django_simple/Dockerfile

```diff
@@ -0,0 +1,80 @@
+ARG PYTHON_VERSION=3.9-slim-buster
+
+# define an alias for the specfic python version used in this file.
+FROM python:${PYTHON_VERSION} as python
+
+# Python build stage
+FROM python as python-build-stage
+
+# Install apt packages
+RUN apt-get update && apt-get install --no-install-recommends -y \
+  # dependencies for building Python packages
+  build-essential \
+  # need git for pip installs
+  git
+
+# Requirements are installed here to ensure they will be cached.
+COPY ./requirements.txt .
+
+# Create Python Dependency and Sub-Dependency Wheels.
+RUN pip wheel --wheel-dir /usr/src/app/wheels  \
+  -r requirements.txt
+
+# Python 'run' stage
+FROM python as python-run-stage
+
+ARG APP_HOME=/app
+
+ENV PYTHONUNBUFFERED 1
+ENV PYTHONDONTWRITEBYTECODE 1
+
+WORKDIR ${APP_HOME}
+
+RUN addgroup --system django \
+    && adduser --system --ingroup django django
```

Comment:
do we need this? why not just root?

Reply:
I've based this on what's generated by https://github.com/pydanny/cookiecutter-django/ so have left some of the "production" configurations as is (like this django user for running the web app). Not necessary of course in our use case. Can do such cleanup.

Reply:
makes sense, do we have it documented somewhere that that is what was used to bootstrap this app?